logstash-core/src/test/java/org/logstash/ClonerTest.java

```diff
@@ -0,0 +1,26 @@
+package org.logstash;
+
+import org.jruby.RubyString;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ClonerTest {
+    @Test
+    public void testRubyStringCloning() {
+        String javaString = "fooBar";
+        RubyString original = RubyString.newString(RubyUtil.RUBY, "fooBar");
```

Comment:
@andrewvc I guess you could use the variable here instead of using the literal twice :)

Reply:
Yep, this is what happens when you code late at night! Fixing